src/vivarium_cluster_tools/psimulate/jobs.py

```diff
@@ -62,8 +62,7 @@ def build_job_list(
     output_root: Path,
     keyspace: branches.Keyspace,
     finished_sim_metadata: pd.DataFrame,
-    make_backups: bool,
-    backup_freq: int,
+    backup_freq: Optional[int],
```

Comment:
Doesn't this make backups off by default? I would assume we'd want them on by default

Reply:
fixed